src/main/java/com/yanolja_final/domain/user/dto/request/EmailRequest.java

```diff
@@ -0,0 +1,21 @@
+package com.yanolja_final.domain.user.dto.request;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class EmailRequest {
+
+    @NotBlank(message = "이메일은 필수 항목입니다.")
+    @Email(message = "올바른 이메일 형식이어야 합니다.")
+    private String email;
+
+    @NotBlank(message = "코드는 필수 항목입니다.")
+    @Size(min = 8, max = 8, message = "코드는 8자여야 합니다.")
+    private String code;
+
+}
```

Comment:
```suggestion
public record EmailRequest(
    @NotBlank(message = "이메일은 필수 항목입니다.")
    @Email(message = "올바른 이메일 형식이어야 합니다.")
    String email, 
    
    @NotBlank(message = "코드는 필수 항목입니다.")
    @Size(min = 8, max = 8, message = "코드는 8자여야 합니다.")
    String code,
 ) {

}
```
자바 17의 record 사용해보시는건 어떠신가요? (참고: https://www.dotnetnote.com/docs/java/records/)
롬복의 `@Data`가 하는 일을 자바에서 알아서 해줍니다!

Reply:
record를 적용은 이상한 곳에서 에러가 자꾸 발생해 일단 보류했습니다...참고해주세요!